packages/core/src/components/viewer-grid/viewer-grid.tsx

```diff
@@ -202,45 +231,51 @@ export class ViewerGrid {
   render() {
     return (
       <Host {...{ [this.mainSize]: true }}>
-        <div class="dso-map-panel" ref={(element) => (this.mapPanel = element)}>
-          <div class="sizing-buttons">
-            <span class="sr-only" aria-live="polite" aria-atomic="true">
-              Breedte tekstpaneel: {this.sizeLabelMap[this.mainSize]}
-            </span>
-            <button type="button" class="shrink" disabled={this.mainSize === "small"} onClick={this.shrinkMain}>
-              <span class="sr-only">Tekstpaneel smaller maken</span>
-              <dso-icon icon="chevron-left"></dso-icon>
-            </button>
-            <button type="button" class="expand" disabled={this.mainSize === "large"} onClick={this.expandMain}>
-              <span class="sr-only">Tekstpaneel breder maken</span>
-              <dso-icon icon="chevron-right"></dso-icon>
-            </button>
-          </div>
-          <div class="main">
-            <slot name="main" />
-          </div>
-        </div>
-        <div
-          id="filterpanel"
-          class="filterpanel"
-          hidden={!this.filterpanelOpen || !this.filterpanelSlot}
+        {this.tabView && (
+          <nav class="dso-navbar">
+            <ul class="dso-nav dso-nav-main">
+              {tabs.map((tab) => (
+                <li class={clsx({ "dso-active": this.activeTab === tab })}>
+                  <button type="button" class="dso-tertiary" onClick={() => (this.activeTab = tab)}>
+                    {this.tabLabelMap[tab]}
+                  </button>
+                </li>
+              ))}
+            </ul>
+          </nav>
+        )}
+        {(!this.tabView || (this.tabView && this.activeTab === "main")) && (
+          <MapPanel ref={(element) => (this.mapPanel = element)}>
+            {!this.tabView && (
+              <div class="sizing-buttons">
+                <span class="sr-only" aria-live="polite" aria-atomic="true">
+                  Breedte tekstpaneel: {this.sizeLabelMap[this.mainSize]}
```

Comment:
@DjangoBlok @LiesbethVorst: Ik neem aan dat we de de sr-only tekst "Breedte tekstpaneel" dan ook willen veranderen in "Breedte hoofdpaneel" ?

Reply:
@LukeVerlaan We hebben een :+1: van Liesbeth, kan jij de opmerking doorvoeren?